engine/src/main/java/com/datatorrent/stram/webapp/StramWebServices.java

```diff
@@ -765,10 +856,10 @@ public JSONObject getPhysicalOperatorProperties(@PathParam("operatorId") int ope
   @GET
   @Path(PATH_LOGICAL_PLAN)
   @Produces(MediaType.APPLICATION_JSON)
-  public JSONObject getLogicalPlan() throws JSONException, IOException
+  public JSONObject getLogicalPlan(@QueryParam("flatten") String flatten) throws JSONException, IOException
```

Comment:
I have done this change as discussed. Removed flatten from parameter and always return flattened dag for modules. and added includeModule parameter to hide show modules from response.